apps/functions-subscription/src/adapters/azure/functions/__tests__/list-trials.test.ts

```diff
@@ -0,0 +1,149 @@
+import { describe, expect, it } from 'vitest';
+import * as TE from 'fp-ts/TaskEither';
+import { makeAValidListTrialRequest, supportHttpRequestHeaders } from './data';
+import { makeFunctionContext, makeTestSystemEnv } from './mocks';
+import { makeListTrialsHandler } from '../list-trials';
+import { aTrial } from '../../../../domain/__tests__/data';
+import { HttpRequest } from '@azure/functions';
+import { Trial, TrialId } from '../../../../domain/trial';
+import { NonEmptyString } from '@pagopa/ts-commons/lib/strings';
+
+describe('makeListTrialsHandler', () => {
+  const anotherTrial = {
+    ...aTrial,
+    id: 'anotherTrialId' as TrialId,
+    name: 'anotherTrialName' as NonEmptyString,
+  } as Trial;
```

Comment:
**suggestion**: cast here is not necessary

you can remove the cast, and also the name (it's a different trial since you are updating the id).

Reply:
cast is redundant somewhere else in this test

Reply:
Removed casts in ce70af07d12fba443dabfbc5aa2b2a899a27949b